tests/testthat/test-pg.R

```diff
@@ -15,207 +15,128 @@
 # 3. Now try connecting again, with the psql command:
 # `psql`
```

Comment:
I think all the text above here can be deleted as it doesn't matter since the pg functions have been gutted.